### PR TITLE
Adds playbooks for New Contributor Content

### PR DESCRIPTION
**Pull request by guineveresaenger**
I put together as best I could the content of a new contributor workshop, as seen at previous KubeCons and similar events.
It is meant to be used as a content playbook.

/sig contributor-experience
/assign @parispittman
/assign @castrojo

### CODE REVIEW

File: events/events-team/content/new-contributor-workshops/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+1. [Kubernetes Orgs Overview](./repo-tour.md)
+1. [Workspace Setup](./workspace-setup.md)
+1. [Local Build&Test](./build-and-test.md)
+1. [Pull Request Exercise](./playground-exercise.md)
+1. [Find Your First Issue](./first-issue-help.md)
+1. [SIG Meet-and-Greet](../sig-contrib-events.md#sig-meet-and-greet)x
+
+## Sample Track for Intermediate Workshop
+
+This workshop differs from the Beginner workshop in that participants are expected to have a working Docker/Kubernetes/Go setup. They have a baisc understanding of how opensource contributions work, and may have opened a PR against Kubernetes before.
```
Comment (geekygirldawn) [inline events/events-team/content/new-contributor-workshops/README.md]: A tiny typo in "basic"

File: events/events-team/content/new-contributor-workshops/build-and-test.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Build and Test
+
+### Duration
+
+40-60 minutes, depending on beginner vs intermediate participants
+
+### Task Overview
+
+Build Kuberentes binaries locally, and do some fun things with them.
```
Comment (geekygirldawn) [inline events/events-team/content/new-contributor-workshops/build-and-test.md]: Typo in "Kubernetes"

File: events/events-team/content/new-contributor-workshops/contributor-paths.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+### Slides
+
+Refer to previous workshops for inspirations
+
+### Details To Cover 
+
+- Who can contribute?
+- What skills are needed?
+- How to Fit Contributions Into Your Job
+- Balance knowlede with learning and the needs of the project
```
Comment (geekygirldawn) [inline events/events-team/content/new-contributor-workshops/contributor-paths.md]: Typo in "knowledge"

File: events/events-team/content/new-contributor-workshops/first-issue-help.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+At this point in the workshop, participants should know what kind of issue they want to tackle.
+
+### Slides
+
+Refer to previous workshops for inspirations
+
+### Details To Cover 
+
+- help-wanted and good-first-issue labels
+- up and coming sandbox projects
```
Comment (geekygirldawn) [inline events/events-team/content/new-contributor-workshops/first-issue-help.md]: Should probably be "Up" to be consistent with other bullets.

File: events/events-team/content/new-contributor-workshops/playground-exercise.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+### Duration
+
+30 - 50 minutes
+
+### Task Overview
+
+Give new contributors the experience of the Kubernetes GitHub workflow!
+This task requires some setup and planning.
+
+Create a New Contributor Workshop folder specific to this event, with teachers' and participants' github handles in the OWNERS file.
```
Comment (geekygirldawn) [inline events/events-team/content/new-contributor-workshops/playground-exercise.md]: Change to "GitHub"

File: events/events-team/content/new-contributor-workshops/new-contributor-workshop-lead.md [hunk 6]
```diff
@@ -30,8 +30,6 @@ Shadows and co-presenters are expected to assist and review these updates.
     - ensure participants sign the CLA
     - ensure attendees meet technical and experience requirements for their course
     - obtain a comprehensive list of names, emails, and github handles
-- Incorporate the new contributor playground into the workshop.
-Create a New Contributor Workshop folder specific to this event, with teachers' and participants' github handles in the OWNERS file - [here is an example](sigs.k8s.io/contributor-playground/seattle)
 - Coordinate with the [SIG Meet and Greet organizer](sig-contrib-events.md) (may be a different role to be determined by core team)
```
Comment (mrbobbytables) [inline events/events-team/content/new-contributor-workshops/new-contributor-workshop-lead.md]:
```suggestion
- Coordinate with the [SIG Meet and Greet organizer](/events/events-team/content/sig-contrib-events.md) (may be a different role to be determined by core team)
```
Fixed link 👍

**Comment by guineveresaenger** [comment]
@mrbobbytables my idea is to have every content presenter make their own slides! this will not make a unified deck, but it will avoid a situation where you're presenting someone else's slides

File: events/events-team/content/new-contributor-workshops/build-and-test.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+### Slides
+
+Clear command line instructions.
+
+### Details To Cover 
+
+- This is the hands-on part. Details are up to you; look to previous workshops for inspiration.
+- Adjust difficulty based on whether this is the Beginner or Intermediate workshop.
+- It is _extremely important_ to run this part of the workshop several times previously, and to make sure that versions match those in the [workspace setup](./workspace-setup.md).
+
```
Comment (alisondy) [inline events/events-team/content/new-contributor-workshops/build-and-test.md]: Add small breaks between sets of instructions for proctors to go and help people with issues, This ensures no one gets left behind.

**Comment by chris-short** [comment]
> @mrbobbytables my idea is to have every content presenter make their own slides! this will not make a unified deck, but it will avoid a situation where you're presenting someone else's slides

I do not like the idea of presenters having to recreate the same or vastly similar content from workshop to workshop. I could see some automation potentially too if there were a standard format, preso, etc. 

I'd like to see a standard Google Slides template applied to modularized content that has copious speaker notes so that presenters can have the information they need to decide how they'd like to present (in the style of their choosing) in one place that a group of maintainers can update as needed.

I definitely appreciate the idea behind not wanting to present someone else's content but, there is a lot of the NCW content that could remain static for the majority of the year.

**Comment by castrojo** [comment]
/lgtm

Not going to block on typos for this, we can always fix up later, nice work. :D

**Pull request merged**